lib/polymer_datepicker.html

```diff
@@ -139,7 +139,7 @@
           <paper-input disabled="[[disabled]]" id="input" on-click="showPicker" required="[[required]]" auto-validate="" value="{{textDate}}" label="[[label]]" on-blur="inputLeft" class$="[[computeInputClass(dateonly)]]"></paper-input>
         </div>
         <div style="position:relative">
-            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-mousedown="cancelClose">
+            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-iron-overlay-closed="doClose" on-mousedown="cancelClose">
```

Comment:
Just push over that branch on github, the PR will get updated.

2015-12-21 15:58 GMT+01:00 barfittc <notifications@github.com>:

> In lib/polymer_datepicker.html
> <https://github.com/dam0vm3nt/polymer_datepicker/pull/2#discussion_r48152891>
> :
>
> > @@ -139,7 +139,7 @@
> >            <paper-input disabled="[[disabled]]" id="input" on-click="showPicker" required="[[required]]" auto-validate="" value="{{textDate}}" label="[[label]]" on-blur="inputLeft" class$="[[computeInputClass(dateonly)]]"></paper-input>
> >          </div>
> >          <div style="position:relative">
> > -            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-mousedown="cancelClose">
> > +            <date-picker-overlay id="pick" style="position:absolute;left:0;top:0" autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]" on-iron-overlay-opened="doPos" on-iron-overlay-closed="doClose" on-mousedown="cancelClose">
>
> It's working. I do not know how to modify my pull request to remove the
> event listener.
> … <#151c50c91ce42a87_>
> On Dec 21, 2015 3:43 AM, "Vittorio Ballestra" <notifications@github.com>
> wrote: In lib/polymer_datepicker.html <#2 (comment)
> <https://github.com/dam0vm3nt/polymer_datepicker/pull/2#discussion_r48125692>>
> : > @@ -139,7 +139,7 @@ > <paper-input disabled="[[disabled]]" id="input"
> on-click="showPicker" required="[[required]]" auto-validate=""
> value="{{textDate}}" label="[[label]]" on-blur="inputLeft"
> class$="[[computeInputClass(dateonly)]]"></paper-input> > </div> > <div
> style="position:relative"> > - <date-picker-overlay id="pick"
> style="position:absolute;left:0;top:0" autofocusdisabled="true"
> autoclosedisabled="true" opened="[[pickerOpen]]"
> on-iron-overlay-opened="doPos" on-mousedown="cancelClose"> > +
> <date-picker-overlay id="pick" style="position:absolute;left:0;top:0"
> autofocusdisabled="true" autoclosedisabled="true" opened="[[pickerOpen]]"
> on-iron-overlay-opened="doPos" on-iron-overlay-closed="doClose"
> on-mousedown="cancelClose"> Could you try that change (remove the event
> listener and replace the binding type) ? I'd prefer binding instead of
> explicit events if it works. I can do it myself but then I would have to
> reject your PR and I would hate it. But if you don't mind I'll do it. —
> Reply to this email directly or view it on GitHub <
> https://github.com/dam0vm3nt/polymer_datepicker/pull/2/files#r48125692>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dam0vm3nt/polymer_datepicker/pull/2/files#r48152891>.
>